### PR TITLE
Task 575

### PR DESCRIPTION
**Pull request by garvitverma**
First set of merges for ingesting DDNA images/cdl files.

### CODE REVIEW

**Comment by garvitverma** [comment]
We should keep this branch alive, since this will get more commits once we get to play with more data types. As the deliveries come.

File: core/templates.yml [hunk 1]
```diff
@@ -89,6 +89,8 @@ keys:
             exr: exr
             jpg: jpeg
             tiff: tiff
+            png: PNG
```
Comment (mbrainerd) [inline core/templates.yml]: why are these uppercase and the others lower? can't we be consistent?

File: core/templates/ingest.yml [hunk 2]
```diff
@@ -1,10 +1,13 @@
+#  copy of /tools/package/sgtk_config/2.5.0/config/core/templates/ingest.yml
 keys:
     ingestImgExtension:
         type: str
         choices:
             exr: EXR
             dpx: DPX
             jpg: JPEG
+            png: PNG
```
Comment (mbrainerd) [inline core/templates/ingest.yml]: Again...consistency in casing please

File: core/templates.yml (hunk 1 above)
Comment (garvitverma) [reply core/templates.yml]: Ah! my bad will fix this.

**Pull request merged**